plugins/httpapi/mso.py

```diff
@@ -150,23 +162,30 @@ def send_request(self, method, path, data=None):
             except (IndexError, TypeError):
                 pass
 
-        if self.params.get("port") is not None:
-            self.connection.set_option("port", self.params.get("port"))
-
-        if self.params.get("username") is not None:
-            self.connection.set_option("remote_user", self.params.get("username"))
-
-        if self.params.get("password") is not None:
-            self.connection.set_option("password", self.params.get("password"))
-
-        if self.params.get("use_proxy") is not None:
-            self.connection.set_option("use_proxy", self.params.get("use_proxy"))
-
-        if self.params.get("use_ssl") is not None:
-            self.connection.set_option("use_ssl", self.params.get("use_ssl"))
-
-        if self.params.get("validate_certs") is not None:
-            self.connection.set_option("validate_certs", self.params.get("validate_certs"))
+        if self.params.get("login_domain") is not None and self.get_option("login_domain") != self.params.get("login_domain"):
```

Comment:
Do we do the same for user/password change?

Reply:
yes that is done below